src/mokka/src/main/java/pl/hycom/mokka/event/StubMappingListener.java

```diff
@@ -0,0 +1,46 @@
+package pl.hycom.mokka.event;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.event.EventListener;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.stereotype.Service;
+import pl.hycom.mokka.emulator.mock.MockConfigurationRepository;
+import pl.hycom.mokka.emulator.mock.model.MockConfiguration;
+
+/**
+ * Implementements ApplicationListener's {@link ApplicationListener}
+ *
+ * Implementation for event listeners
+ *
+ * @author adam.misterski@hycom.pl
+ */
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class StubMappingListener {
+
+    private final WireMockServer wireMockServer;
+
+    private final ConversionService conversionService;
+
+    private final MockConfigurationRepository mockConfigurationRepository;
+
+    StubMapping stubMapping = new StubMapping();
+
+    @EventListener({StubMappingRemoveEvent.class, StubMappingDisabledEvent.class})
+    public void removeStubMapping(StubMappingEvent event){
+        stubMapping.setId(event.getUuid());
+        wireMockServer.removeStubMapping(stubMapping);
+    }
+
+    @EventListener({StubMappingAddEvent.class, StubMappingEnabledEvent.class})
+    public void addStubMapping(StubMappingEvent event){
+        MockConfiguration mockConfiguration = mockConfigurationRepository.findById(event.getId()).get();
+        wireMockServer.addStubMapping(conversionService.convert(mockConfiguration, StubMapping.class));
```

Comment:
Could you please add log about added stubMapping?
Also it may happen that `findById` will return nothing (`Optional.empty`). Would be good to check it and react properly (log warning, skip conversion)